public/index.html

```diff
@@ -2,14 +2,14 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="theme-color" content="#343434" />
     <meta
       name="description"
       content="Displays the data in the Philanthropy Data Commons pilot. Read more at philanthropydatacommons.org."
     />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <meta name="theme-color" content="#343434" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png">
```

Comment:
Did you mean to remove the ` /` in this `<link>`?

Reply:
Ooh, I did not! Jason those are some eyes. Well worth the fix and force-push. I say… genuinely. (This would bother me so much to discover later.)

Reply:
Okay fixed, along with the un-closed SVG `<link>` tag. Thanks!